pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/exceptions/InvalidInputException.java

```diff
@@ -16,14 +16,12 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.tools.tuner.query.src.stats.wrapper;
+package org.apache.pinot.controller.recommender.io.exceptions;
```

Comment:
Why did we move this class to the recommender? Is it not being used elsewhere?

Reply:
The tuner package was the old code for index recommender. I did reuse some of the old code, so github shows I'm moving some files from the old package. But I'm not sure why this happens in InvalidInputException class as it is completely new.

Reply:
Got it. Thanks